### PR TITLE
refactor(edgeless): split toolbar modules

### PR DESCRIPTION
**Pull request by alt1o**
https://user-images.githubusercontent.com/21084335/220945887-c7faedb2-c912-4728-9c69-9dfca2ca6897.mov

### CODE REVIEW

File: packages/blocks/src/page-block/edgeless/toolbar/shape-tool-button/shape-tool-button.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+@customElement('edgeless-shape-tool-button')
+export class EdgelessShapeToolButton extends LitElement {
+  @property()
+  mouseMode?: MouseMode;
+
+  @property()
+  edgeless?: EdgelessPageBlockComponent;
+
+  private _shapesMenu?: ShapesMenuInstance;
```
Comment (doodlewind) [inline packages/blocks/src/page-block/edgeless/toolbar/shape-tool-button/shape-tool-button.ts:50]: We prefer explicit `null` as the void value.

**Comment by doodlewind** [comment]
TODOs:

* Try remove the `edgeless-toolbar-with-flag` wrapper and attach edgeless-toolbar without imperative `document.createElement`

**Review by Himself65 — COMMENTED** [review]
I know what you are doing, but the first left icon is `quare`

**Review by doodlewind — APPROVED** [review]
LGTM, cheers!

**Pull request merged**